templates/contact/index.html

```diff
@@ -11,58 +11,117 @@
 {% block content %}
 <div id="main-content" class="inner-wrapper">
 
-<section class="p-strip is-bordered is-deep">
+<section class="p-strip is-shallow">
   <div class="row">
-    <div class="col-8">
-      <h1>联系Ubuntu</h1>
-    </div>
-  </div>
-
-  <div class="row">
-    <div class="col-5">
-      <h2>商业合作咨询</h2>
-
-      <ul class="p-list--divided">
-
-  <li class="p-list__item">物联网及设备 <a class="u-float--right p-link--external" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn-contact-iot', 'eventValue' : undefined });">我要咨询</a></li>
-
-
-
-  <li class="p-list__item">云及服务器 <a class="u-float--right  p-link--external" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn-contact-cloud', 'eventValue' : undefined });">我要咨询</a></li>
-
-
-</ul>
-
-
-      <h2>获取支持</h2>
-      <p>
-        向我们<a href="https://pages.ubuntu.com/contact-us-in-chinese.html">咨询</a>我们的产品，支持，培训或咨询，我们将尽快与您联系。
-      </p>
-      <p>
-        如您对Ubuntu中文官网有任何意见或需修正的地方，可到Github<a href="https://github.com/canonical-websites/cn.ubuntu.com/issues/new">提交</a>bug。.
-      </p>
-    </div>
-
-    <div class="col-7 col-start-1">
-      <h2>技术协助</h2>
-      <p>
-        如您正在寻找Ubuntu的支持，<a href="https://www.ubuntu.com/support">我们的支持团队</a>可以帮助您。
-      </p>
-
-      <h2>新闻媒体</h2>
-      <p>
-        记者或分析机构如想获取相关新闻信息请邮件至<a href="mailto:chinamarketing@canonical.com">chinamarketing</a>。很遗憾，我们无法回复非传媒或分析师的查询。
-      </p>
-
-      <h2>法律查询</h2>
-      <p>
-        请注意，我们无法向您提供法律建议．对于其他任何关于Ubuntu的法律疑问，请联系我们<a href="mailto:legal@canonical.com">legal@canonical.com</a>。
-      </p>
-
-      <h2>商标咨询</h2>
-      <p>
-        如您要申请Ubuntu商标许可或进行其他商标查询，请<a href="https://www.ubuntu.com/legal/terms-and-policies/contact-us">提交查询</a>。我们的商标团队成员很快会与您联系。
-      </p>
+     <h1>获取支持</h1>
+        <div class="col-5">
+          <h4>技术协助</h4>
+              <p>如您正在寻找Ubuntu的支持，<a href="https://www.ubuntu.com/support">我们的支持团队</a>可以帮助您。</p>
```

Comment:
I missed this first time when I looked at it. Can you please make sure you use 2 spaces for indentation and not 4 to be inline with our best practices https://canonical-web-and-design.github.io/practices/coding/html.html

Reply:
OK, I didn't know there's kind of guide for code indentation, thank for the information. Fixed with 2 spaces indentation.